### PR TITLE
feat: require authorized user for direct upload

### PR DESCRIPTION
**Pull request by martintomas**
Making sure that `/rails/active_storage/direct_uploads` route can be accessed only by authenticated users

## Testing instructions

Can be tested same way as other API endpoints via Rswag documentation

## Tracking

https://vizzuality.atlassian.net/browse/LET-336

### CODE REVIEW

**Comment by tsubik** [comment]
Ahh, right, when requiring CSRF token, probably there will be problems with using npm library for activestorage on frontend side.

**Comment by tsubik** [comment]
Let's make sure there will be no problems on the frontend, if there are maybe we can just one security gate like require json and not require CSRF token. Then you can inherit from ActiveStorage Base controller.

**Comment by tsubik** [comment]
Lol, that library is pretty rails specific https://github.com/rails/rails/blob/main/activestorage/app/assets/javascripts/activestorage.js#L520

**Comment by tsubik** [comment]
Maybe then frontend can save csrf-token in meta attribute for activestorage to pick it up.

**Comment by martintomas** [comment]
Tried couple examples with `ActiveStorage::DirectUploadsController` as parent for `DirectUploadsController ` and was not able to make `Devise` work properly, therefore I will keep this reimplementation version of code for now. Definitely feel free to tell me if you have some idea how make this better :)

**Comment by martintomas** [comment]
Oki douky, moved authorization directly to routes which made it work and does not require to patch or reimplement `ActiveStorage::DirectUploadsController`

**Review by agnessa — APPROVED** [review]

File: backend/config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   get :health_check, to: ->(_env) { [204, {}, [""]] }
 
+  authenticate :user do
```
Comment (tsubik) [inline backend/config/routes.rb:4]: Nice!

**Review by tsubik — APPROVED** [review]

**Pull request merged**